torch/nn/modules/padding.py

```diff
@@ -89,8 +89,8 @@ class ConstantPad2d(_ConstantPadNd):
             :math:`\text{padding\_right}`, :math:`\text{padding\_top}`, :math:`\text{padding\_bottom}`)
 
     Shape:
-        - Input: :math:`(N, C, H_{in}, W_{in})`
-        - Output: :math:`(N, C, H_{out}, W_{out})` where
+        - Input: :math:`(*, C, H_{in}, W_{in})` where :math:`*` is one dimension or none.
```

Comment:
As discussed in the `AdaptiveAvgPool1d` PR, I think it's more readable to list out the two possibilities linked by "or" for the input / output shapes

tiny nit: comma after shape and before "where" throughout for consistency with other docs